manual/en/docs/flow/actions/random_element_action.md

```diff
@@ -0,0 +1,25 @@
+# Return random item plugin
+
+## Input
+This plugin takes any payload object as input.
+
+## Output
+This plugin outputs a random value from provided list, accessible by key **value**.
+
+## Configuration
+
+#### With form
```

Comment:
You do not have to document the Form - the form should be described in the Form good enough for the user to understand it.